frontend/public/components/pdb/PDBForm.tsx

```diff
@@ -0,0 +1,308 @@
+              <TextInput
+                isRequired
+                type="text"
+                name="name"
+                aria-labelledby={t('public~name')}
+                placeholder="example"
+                value={formValues.name}
+                onChange={handleNameChange}
+                isDisabled={!!existingResource}
+              />
+            </FormGroup>
+          </GridItem>
+          <GridItem span={12}>
+            <FormGroup
+              label={t('public~Labels')}
+              isRequired
+              fieldId="labels"
+              labelIcon={
+                <FieldLevelHelp>
+                  <p className="co-section-heading-tertiary">
+                    <strong>Selector</strong>
+                  </p>
+
+                  <p>
+                    Label query over pods whose evictions are managed by the disruption budget. A
+                    null selector will match no pods, while an empty ({}) selector will select all
+                    pods within the namespace.
+                  </p>
+                </FieldLevelHelp>
+              }
+            >
+              <SelectorInput
+                onChange={(l) => handleSelectorChange(l)}
+                tags={SelectorInput.arrayify(formValues.selector)}
+                labelClassName="labelClassName"
+                autoFocus
+              />
+
+              {matchingSelector && (
+                <Alert
+                  className="co-alert"
+                  variant="warning"
+                  title={t('public~ {{labels}} is already covered by another PodDisruptionBudget', {
+                    labels,
+                  })}
+                  isInline
+                />
+              )}
+            </FormGroup>
+          </GridItem>
+          <FormGroup
+            fieldId="requirement"
+            isRequired
+            label={t('public~Availability requirement')}
+            labelIcon={
+              <FieldLevelHelp>
+                <p className="co-section-heading-tertiary">
+                  <strong>Availability Requirement</strong>
+                </p>
+
+                <p>
+                  <strong>minAvailable </strong>
+                  An eviction is allowed if at most "maxUnavailable" pods selected by "selector" are
+                  unavailable after the eviction, i.e. even in absence of the evicted pod.
+                </p>
+
+                <p>
+                  <strong>minAvailable </strong>
+                  An eviction is allowed if at least "minAvailable" pods selected by "selector" will
+                  still be available after the eviction.
+                </p>
+                <p>
+                  {t('public~More information:')}&nbsp;
+                  <ExternalLink
+                    href="https://docs.openshift.com/container-platform/4.9/rest_api/policy_apis/poddisruptionbudget-policy-v1.html#poddisruptionbudget-policy-v1"
```

Comment:
You need to use `openshiftHelpBase` to link to the right edition and version of the docs.

Reply:
It looks like the OKD docs have different paths, so we'd need to check `isUpstream()` and find the right document for upstream OKD. @yselkowitz can you help?